_i18n/en/resources/user-guides/mine-to-pool.md

```diff
@@ -114,3 +201,21 @@ You need to edit your system's group policies to enable locking large pages. Her
 7. In the Local Security Setting – Lock pages in memory dialog box, click Add User or Group.
 8. In the Select Users, Service Accounts, or Groups dialog box, add an account that you will run the miner on
 9. Reboot for change to take effect.
+
+#### Allowing large pages on Linux
+
+Firstly stop the miner (if it's running), run the following commands to enable
+large pages and then start the miner as root:
+
+	sudo sysctl -w vm.nr_hugepages=1168
+	sudo ./xmrig
+
+You may have to increase 1168 depending on how many NUMA nodes your
+CPU(s) have.
+
+## Getting Help
+
+An active Monero mining community on Reddit is
+[/r/MoneroMining](https://www.reddit.com/r/MoneroMining/). You can
```

Comment:
We don't link to /r/moneromining from getmonero. See: https://repo.getmonero.org/monero-project/monero-site/-/merge_requests/1195. Better point to /r/monerosupport or whatever.